ui/app/AppLayouts/Communities/controls/Options.qml

```diff
@@ -24,7 +26,7 @@ Column {
         id: archiveSupport
 
         width: parent.width
-        height: d.optionHeight
+        height: visible ? d.optionHeight : 0
```

Comment:
I think it should be implicitHeight instead of height, without setting height to 0

Reply:
<img width="628" alt="Screenshot 2023-07-21 at 16 41 10" src="https://github.com/status-im/status-desktop/assets/2522130/4f5df17c-7cd3-4fe0-be96-29cbf3403e46">
For some reason it does not work properly